src/core/packages/deprecations/server-internal/src/routes/post_validation_handler.ts

```diff
@@ -49,6 +55,9 @@ export function createRouteDeprecationsHandler({
       const client = coreUsageData.getClient();
       // no await we just fire it off.
       void client.incrementDeprecatedApi(counterName, { resolved: false });
+
+      const { message, meta } = getEcsResponseLog(ensureRawRequest(req), logger);
+      logger.debug(message, meta);
```

Comment:
```suggestion
      if (logger.isLogLevelEnabled('debug') {
          const { message, meta } = getEcsResponseLog(ensureRawRequest(req), logger);
          logger.debug(message, meta);
       }
```

I know this isn't the hottest code path, but I can see how if users are hitting deprecated endpoints a lot we might be doing unnecessary work without this log level enabled check.

Reply:
It's other use is over here (for example):

https://github.com/elastic/kibana/blob/8bb48a44275a73187f8330bd2c3ad12e6c8edb61/src/platform/packages/shared/kbn-analytics/src/reporter.ts#L76